src/components/button/css.module.scss

```diff
@@ -13,3 +13,38 @@
   cursor: pointer;
   border-radius: 50%;
 }
+
+.roundedEdges{
+  margin: 0;
+  padding: 0;
+  height: 40px;
+  width: 40px;
+  background-color: var(--primary-color);
+  border: none;
+  color: white;
+  text-align: center;
+  text-decoration: none;
+  display: block;
+  // margin: 0 4px;
+  cursor: pointer;
+  border-radius: 10px;
+  
+}
+
+.inner{
+  display: inline-block;
+}
+
+
+
+#changeSeconds{
```

Comment:
Why are you using this?

Reply:
I've adjusted the come somewhat from before this pull request, so the lines don't match up as much with my code anymore, but if you mean the roundedEdges class, it's to make the + and - buttons which are square with rounded edges